docs/source/integrations/deltalake_versioning.md

```diff
@@ -0,0 +1,107 @@
+
+This tutorial explores how to use Delta tables in your Kedro workflow and how to leverage the data versioning capabilities of Delta Lake.
+
+## Prerequisites
+
+In this example, you will use the `spaceflights-pandas` starter project which has example pipelines and datasets to work with. If you haven't already, you can create a new Kedro project using the following command:
+
+```bash
+kedro new --starter spaceflights-pandas
+```
+
+Kedro offers various connectors in the `kedro-datasets` package to interact with Delta tables: [`pandas.DeltaTableDataset`](https://github.com/kedro-org/kedro-plugins/blob/main/kedro-datasets/kedro_datasets/pandas/deltatable_dataset.py), [`spark.DeltaTableDataset`](), [`spark.SparkDataset`](), [`databricks.ManagedTableDataset`](), and [`ibis.FileDataset`]()ß support the delta table format. In this tutorial, we will use the `pandas.DeltaTableDataset` connector to interact with Delta tables using Pandas DataFrames. To install `kedro-datasets` alongwith dependencies required for Delta Lake, add the following line to your `requirements.txt`:
+
+```bash
+kedro-datasets[pandas-deltatabledataset]
+```
+
+Now, you can install the project dependencies by running:
+
+```bash
+pip install -r requirements.txt
+```
+
+## Using Delta tables in catalog
+
+To use Delta tables in your Kedro project, you can update the `base/catalog.yml` to use `type: pandas.DeltaTableDataset` for the datasets you want to save as Delta tables. For this example, let us update the `model_input_table` dataset in the `base/catalog.yml` file:
+
+```yaml
+model_input_table:
+  type: pandas.DeltaTableDataset
+  filepath: data/02_intermediate/model_input_table
+  save_args:
+    mode: overwrite
+```
+
+You can specify `save_args` to specify the mode of saving the Delta table. The `mode` parameter can "overwrite" or "append" depending on whether you want to overwrite the existing Delta table or append to it. You can also specify [additional saving options that are accepted by the `write_deltalake` function in the `delta-rs` library](https://delta-io.github.io/delta-rs/python/api_reference.html#writing-deltatables) which is used by `pandas.DeltaTableDataset` under the hood.
+
+When you run the Kedro project with `kedro run` command, the Delta table will be saved to the location specified in the `filepath` argument as a folder of `parquet` files. This folder also contains a `_delta_log` directory which stores the transaction log of the Delta table. Subsequent runs of the pipeline will create new versions of the Delta table in the same location and new entries in the `_delta_log` directory.
+
+To load a specific version of the dataset, you can specify the version number in the `load_args` parameter in the catalog entry:
```

Comment:
I think this deserves its own section title, to be easier to locate?

Reply:
I made it a subsection within "Using delta tables in catalog"